src/platforms/windows/daemon/wireguardutilswindows.cpp

```diff
@@ -80,32 +80,28 @@ QList<WireguardUtils::PeerStatus> WireguardUtilsWindows::getPeerStatus() {
 }
 
 bool WireguardUtilsWindows::addInterface(const InterfaceConfig& config) {
-  QString tunnelFile = WindowsCommons::tunnelConfigFile();
-  if (tunnelFile.isEmpty()) {
-    logger.error() << "Failed to choose the tunnel config file";
-    return false;
-  }
-
   QStringList addresses;
   for (const IPAddress& ip : config.m_allowedIPAddressRanges) {
     addresses.append(ip.toString());
   }
 
   QMap<QString, QString> extraConfig;
   extraConfig["Table"] = "off";
-  if (!WgQuickProcess::createConfigFile(tunnelFile, config, extraConfig)) {
+  QVariant configString =
+      WgQuickProcess::createConfigString(config, extraConfig);
+  if (!configString.isValid()) {
     logger.error() << "Failed to create a config file";
     return false;
   }
 
-  if (!m_tunnel.start(tunnelFile)) {
+  if (!m_tunnel.start(configString.toString())) {
     logger.error() << "Failed to activate the tunnel service";
     return false;
   }
 
   // Determine the interface LUID
   NET_LUID luid;
-  QString ifAlias = QFileInfo(tunnelFile).baseName();
+  QString ifAlias = "MozillaVPN";
```

Comment:
You can get this from `WireguardUtils::interfaceName()` which should reduce the number of hard-coded strings floating around.